common/repo/GitRepo.java

```diff
@@ -153,14 +154,11 @@ public void tagHead(String name) {
     CommandResult commandResult = runCommand("status --porcelain");
     ImmutableList<String> lines = splitLines(commandResult.stdout);
     for (String line : lines) {
-      line = line.replaceFirst("\\s{2}", " ");
+      line = line.replaceFirst("  ", " ");
       String[] parts = line.trim().split(" ");
-
-      File.Builder fileBuilder = File.newBuilder();
-      fileBuilder.setAction(getAction(parts[0]));
-      fileBuilder.setFilename(
-          fileBuilder.getAction().equals(File.Action.RENAME) ? parts[3] : parts[1]);
-      files.add(fileBuilder.build());
+      Action action = getAction(parts[0]);
+      String filename = action.equals(File.Action.RENAME) ? parts[3] : parts[1];
```

Comment:
Can you add an example of what a regular line and a rename line looks like?
```
// We extract the action and filename from line. Here are some example lines:
// <regular line>
// <rename line>
```

Reply:
Done.